test-runner-poc/config.js

```diff
@@ -66,8 +69,8 @@ class NativeConfiguration {
   url(username, password) {
     const url = this.options.url || 'mongodb://%slocalhost:27017/' + this.db;
     // Fall back
-    const auth = username && password ? f('%s:%s@', username, password) : '';
-    return f(url, auth);
+    const auth = username && password ? String(username) + ':' + String(password) + '@' : '';
```

Comment:
Can we use the syntax of `${username}:${password}@''`?

Reply:
fixed!